docgen.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# TODO: gendoc CI on push?
+import re
+
+def strip_head(s):
+    return s.strip().lower().replace("`", "").replace("*", "").replace(" ", "-")
+
+with open('./norg_tutorial.norg') as file:
+    lines = file.read().strip()
+
+headings = re.findall(r"^\s*(\*+ [a-zA-Z0-9_\-\.\ :`\/\+\(\)]+)", lines, re.MULTILINE)
+
+res = ""
+for head in headings:
+    res += f"norg-tutorial-{strip_head(head)}\t../norg_tutorial.norg\t/{head}\n"
```

Comment:
I think we need to only write the first line in `{head}`?

`re.findall` will find multiline headers (from new norg spec) right?

Reply:
OK.

Well to be exact, we need to exclude headers inside `|example` and so on and so on, but it'll do the job for now ig.

And I don't plan to use any multiline header in this tutorial so it'll be OK.